### PR TITLE
Refactoring MetaClient Exceptions

### PR DESCRIPTION
**Pull request by Marcosrico**
### Issues

- [x] My PR addresses the following Helix issues and references them in the PR description:

N/A

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

Moving exception classes into a separate package: org.apache.helix.metaclient.exception

### Tests

- [ ] The following tests are written for this issue:

(List the names of added unit/integration tests)

- The following is the result of the "mvn test" command on the appropriate module:

(If CI test fails due to known issue, please specify the issue and test PR locally. Then copy & paste the result of "mvn test" to here.)

### Code Quality

- My diff has been formatted using helix-style.xml 
(helix-style-intellij.xml if IntelliJ IDE is used)

### CODE REVIEW

File: meta-client/src/main/java/org/apache/helix/metaclient/impl/zk/util/ZkMetaClientUtil.java [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 import org.apache.helix.metaclient.api.MetaClientInterface;
 import org.apache.helix.metaclient.api.OpResult;
-import org.apache.helix.metaclient.constants.*;
+import org.apache.helix.metaclient.exception.*;
```
Comment (desaikomal) [inline meta-client/src/main/java/org/apache/helix/metaclient/impl/zk/util/ZkMetaClientUtil.java:24]: nit: typically this is not a good practice but for now, don't change as it was just existing code.

**Review by desaikomal — APPROVED** [review]
LGTM. thanks @Marcosrico

**Review by qqu0127 — APPROVED** [review]
LGTM

**Comment by Marcosrico** [comment]
This PR is ready to merge, approved by @qqu0127 and @desaikomal 
Commit message:
Refactoring MetaClient Exception. Creating separate exception directory.

**Pull request merged**